### PR TITLE
New contributor victor

### PR DESCRIPTION
**Pull request by vbello-tech**
Organizations.

### CODE REVIEW

**Comment by vbello-tech** [comment]
New changes to organisation.

File: mediq/settings.py [hunk 1]
```diff
@@ -48,9 +48,11 @@
 
     # third-party apps
     'rest_framework',
+    'rest_framework_simplejwt',
 
     # mediq apps
     'accounts',
+    'organizations',
```
Comment (ourcardinalhealth) [inline mediq/settings.py:55]: hi @vbello-tech 
let's use british english across the app. "s" instead of "z"
oraganisation, instead of organization

Comment (ourcardinalhealth) [reply mediq/settings.py:55]: ignore this. i can see you've used that across the entire application. we'll stick with american english

**Review by ourcardinalhealth — CHANGES_REQUESTED** [review]
you have reused an old branch for this pr
this pr should be created under a new branch the describes what you've worked on

**Comment by vbello-tech** [comment]
I will be closing this pr because it was meant for another branch that i have created.

**Pull request closed without merging**